lib/eco/cflexer/lexer.py

```diff
@@ -189,13 +194,14 @@ def find_next_token(self):
                         lookahead = 1
                         break
                 source = self.text[start: ]
-                result = self.make_token(start, self.last_matched_state, source, lookahead = lookahead)
+                result = self.make_token(start, -1, self.last_matched_state, lookahead = lookahead)
                 self.last_matched_index = start + len(source)
                 self.adjust_position(source)
                 if self.ignore_token(self.last_matched_state):
```

Comment:
Ah this was just me testing if these parts are ever reached, which I guess they are not, so I should remove them all together.